<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paintDialog</class>
 <widget class="QDialog" name="paintDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>parameter</string>
   </property>
   <widget class="QRadioButton" name="vertical_radioBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dot_line_cBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="numSelect_spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="Horizontal_rationBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QComboBox" name="color_cBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="YvalueEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="NumSpinbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Operation</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>178</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>8</number>
     </property>
     <item row="7" column="1">
      <widget class="QPushButton" name="yPhase_Btn">
       <property name="text">
        <string>yPhase</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="Red_Btn">
       <property name="text">
        <string>Red</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="xPhase_Btn">
       <property name="text">
        <string>xPhase</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="Green_Btn">
       <property name="text">
        <string>Green</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="Blue_Btn">
       <property name="text">
        <string>Blue</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="XY_Btn">
       <property name="text">
        <string>XY_value</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="valueBtn">
       <property name="text">
        <string>value</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="clear_Btn">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
